templates/circleci-2.0/config.yml

```diff
@@ -152,3 +152,6 @@ workflows:
       - run-behat-tests
       - run-code-sniffer
       - run-code-coverage
+      - run-code-coverage:
```

Comment:
Looking at https://circleci.com/docs/2.0/workflows/#sequential-job-execution-example it seems that we should remove the above line, right?